### DISCUSSION TITLE
Announcement: microCOVID needs your help

### DISCUSSION THREAD

**Discussion by apiology** [microCOVID/microCOVID#1626] [General]
COVID and our world's infrastructure around it continue to change, but microCOVID and the assumptions it was built on have not.  I personally think having a tool which is even just directionally correct and reinforces smartly targeted safety habits is still very useful.  However, I think that crossing the line to being actively misleading on risk is a real danger.

If you agree, we need your help to turn this around.  Thoughtful contributions are very welcome.  There's a wide variety of things that can be worked on, but I tried to narrow it down to what I think the most impactful things we could do.

Please let me know what you think and what I can do to help you help.

**Steps we need help from the community on:**

- Stabilize and monitor data pipeline
- Modernize prevalence input
- Modernize boosters in model

If we aren’t able to make these these things happen, my feeling is that we ought to go ahead and take the next step in the site's deprecation by, say, **Feb 1**.

## Stabilize and monitor data pipeline

Our data pipeline has gone down for days and weeks with no attention when upstream sources make changes or have data issues. Keeping up with these issues has been my first focus. I think I’ve made significant progress here, but some things remain to do. For instance, right now, those pipeline alerts go to one otherwise busy person. Getting something which would create GitHub issues on failure and/or send alerts onto our contributor Slack would be very helpful.

Some locations are no longer providing test data at all. Quite a few provide it less reliably and less frequently. Our pipeline needs to be robust against both issues. Our model needs to ensure that missing data is mitigated as possible. Our UI needs to communicate the impact of missing data clearly to users, perhaps by gathering some kind of prevalence assessment from the user directly.

A validation procedure will also be very helpful, to characterize the impact of changes made by folks like me who didn’t write the original code and model. I don’t know exactly what this looks like yet, but something pragmatic we can build on is better than nothing.

### Modernize prevalence input

Once we have existing functionality shored up, we need to start modernizing. Our model uses local prevalence (as measured by officially reported testing numbers) to adjust risk. The concept of adjusting your activities and safety measures based on risk is a really compelling microCOVID feature. That said, we know the amount of official testing has reduced, as a secular trend independent of the cyclical waves of infections. Personally, I haven’t seen a line in my neighborhood for the testing kiosks for a while now. The milder acute symptoms of Omicron strains, the popularization of home testing and a general weariness of safety measures are surely causes of the trend towards underreporting.

We should find a way to make an adjustment to try to correct for this, and ensure we are expressing to users the limitations of this good faith measure along the way. Again, I lean towards the pragmatic.

### Modernize boosters in model

Our vaccination model doesn’t model the short term benefit of boosters on immunity.  Ben Shaya has researched [how to update this part of the model](https://paper.dropbox.com/doc/Vaccine-Decay-vdYODjg0Sbeu6bpl3Uxic) based on [a paper on neutralizing antibodies](https://www.nature.com/articles/s41591-021-01377-8.pdf); we should get this work out for review from anyone who is up for it and get it into the site model absent any blockers identified.

### Deprecate site if not able to modernize by Feb 1

The above is a big ask; I want to be clear that I’m unlikely to be able to do this work solo. If we don’t reach this level of improvement, we should have a plan for next steps. Maybe that’s turning off prevalence updates (which will surely die absent maintenance) and then leaving the site up as-is with an updated banner making it clear it is historical in nature and should not be used for current risk assessment.

What do you think?

**Comment by zerotrickpony** [comment]
I really appreciate you laying this out. I will try to make some time to help in the coming weeks. I can code, but the modeling statistics are probably a bit over my head, so it would be great if a medical / stats person could figure out at least in principle how to better model prevalence in our new world of under-reporting. The big value to me of microcovid is the well-researched quantified model.

I like with the idea of updating the UX with permanent-ish warnings that address the hazards from non-reporting jurisdictions and failed pipeline runs. Sounds affordable and inevitable.

Unfortunately data pipeline stability seems to me like the hardest task and in a sense the least valuable. I bet various jurisdictions will continue to slow down or drop out of the reporting even aside from the reality that the reported case counts were always a grab bag of biases. Even if we stabilize the data ingestion perfectly, microcovid.org will still be missing a lot of localities.

It might be interesting to survey popular jurisdictions and see if there's an easy way to encourage users to manually input their local prevalence? To pick one strange example, in Northern Ireland they've dispensed with dashboards entirely, BUT the department of health puts out a weekly report of a comparatively well controlled prevalence study. ([example](https://www.health-ni.gov.uk/articles/covid-19-infection-survey)) The data quality seems great but the numbers are buried in a random PDF with no history. I'd despair of ingesting them automatically, but a determined local could find this for themselves and input manually. I wonder how many other localities have a situation like this.

**Reply by apiology** [reply]
Note that we do have a user-facing warning about failed pipeline runs: https://github.com/microCOVID/microCOVID/pull/1437

Agreed that manual prevalence input would be an improvement when we have no automatic data for a locale.  Another complementary idea I've had is to use the numbers from the containing locale (e.g., in the US, use state data instead of county data, etc).  Would need to make sure it is clear in the UI that this is happening, of course.

**Reply by apiology** [reply]
More specifically to @zerotrickpony's idea:

If there are enough of these situations not covered by our pipelines to form a pattern worth acting on, maybe folks could file issues with the examples?  If the benefit would be enough to be worth the code complexity and effort, someone could move forward with a manual prompt with those as the input.

**Comment by Gradstudent134** [comment]
Hi there, I just wanted to say that my household uses the microcovid tool very actively several times a week. Thanks for your service. Unfortunately none of us have many free volunteer hours, I could donate about two hours a month to reviewing / being a second set of eyes on written material. We would be interested to support the project Patreon style if that was possible, to financially support developers’ time on the tool. 

Please keep Microcovid up if it’s possible, it’s helping us make good risk decisions! Thank you again.

**Reply by apiology** [reply]
Thanks for your kind words!  

As a practical matter, it looks like the current donations link goes via Paypal to @catherio.  She's no longer active on the project, and I haven't been in touch with her.  If someone wanted to, they could try to get in touch with her to try to see if there's any balance that might be available.   If they think this is likely to bear fruit, they could figure an updated mechanism for folks to support financially, publicize this and perhaps brainstorm ways to turn that into site improvement.

**Comment by aidantjones** [comment]
I'm very interested in becoming involved in this project. Over the past two years, I've relied upon microCOVID and it's given me a lot of comfort.

I just so happens that around the same time that the pandemic started I began learning front-end development. It looks like the back end is written mostly in Python and the front-end in React? Unfortunately, I'd have to learn both of those items. But I should anyway and am willing to learn. I also come from a social science background and could work on interpreting literature for the purposes of the microCOVID tool.

I agree with the goals you outlined except for the deprecation date. I feel that if the current model is significantly outdated enough to provide potentially dangerous misleading information that it's imperative that at least site messaging be updated ASAP. We're already in holiday season and travel is supposed to be more prevalent than in previous years. An outdated tool could jeopardize the health of its users.

**Reply by apiology** [reply]
Thanks for your offer of help.  There's no traditional backend, but the React site is redeployed daily with fresh hard-coded data that is generated by a Python script.

Yes, I agree that the alert should go up ASAP and hope my post didn't suggest otherwise.  Happy to help wordsmith the warning if someone wants to take a crack at it; I had in mind a short warning and a pointer to this discussion page.  There's an Alert component that we have used in the past for this sort of thing:

https://github.com/microCOVID/microCOVID/blob/main/src/pages/Calculator.tsx#L241-L256

**Comment by zdamien** [comment]
Couple thoughts:

1) Assume that covid deaths are more accurate now, and use that as a proxy for cases, like cases being 500-2000x the death rate. Of course, the death rate is actually a fraction of the case rate from a few weeks ago, so it's a crude heuristic especially when infections rise or fall. But still, could be a useful option at least.

1a) Or apply a x10 or x20 correction to official case rates. This shouldn't be done everywhere, though; I would guess that at least a few East Asian countries have widespread official testing, since they have much higher 'case rates' than other countries but similar death rates.

2) If using the official transmission rate, emphasize that the risk estimates are a **minimum**.  A lot of the time, even the official case rates are still too high for me to want to eat out in restaurants, even without applying a x10 or x20 correction factor.

**Reply by apiology** [reply]
Yeah, I've had similar thoughts.  I think it'd be great to get ideas like these into dedicated issue(s), even more so if we can come up with at least defensible rationales for the constant factors.

**Comment by apiology** [comment]
Thanks for the thoughts, everyone!

I'll try to do my part by watching for PRs to push through from folks.

In the meantime, a first area you could help with would be to break these ideas apart into particular issues to be filed, where we can dive into them more deeply.  There's a lot to get into here.

**Comment by tmadway** [comment]
Has wastewater monitoring data been considered to help approximate case rates, & if not is there a reason it can't be used?

I don't have much coding capacity rn but do have some thoughts on the structure & wording of input options that I'll try to compile; eg ability to specify quality of air filtration, further differentiation between type & fit of masks (ie fit tested vs looks to fit vs visibly gaping, also addition of P100 elastomerics), more specificity re physical proximity (ie unsure how to input carpooling), updated risk profiles (or maybe way to roughly calculate any profile), etc

I've really appreciated this tool, thank you so much not making it! Hoping that once updated I'll be able to reference it again to help loved ones visualize why different precautions are important / certain situations are significantly riskier than others...

**Reply by apiology** [reply]
I've heard wastewater data referenced by an inactive contributor as a potential lead.  Further investigation would be very welcome; an issue where folks can collaborate with ideas would be a good start.

**Reply by turtlecrk** [reply]
Here is a link to the New York wastewater testing program, which is just starting up:
https://storymaps.arcgis.com/stories/b30ca571e00b42779875e3eaf7577b1b
Boston started wastewater testing in March 2020:
https://www.mwra.com/biobot/biobotdata.htm

As PCR testing winds down, this probably will be the best indicator for local risks.

**Reply by piratewendy99** [reply]
> I've heard wastewater data referenced by an inactive contributor as a potential lead. Further investigation would be very welcome; an issue where folks can collaborate with ideas would be a good start.

I would second the vote to look to wastewater, as that data wont "lie" and the infrastructure for wastewater monitoring seems to be expanding.  Two places where I follow wastewater trends are:
https://biobot.io/data/ 
and
https://covid.cdc.gov/covid-data-tracker/#wastewater-surveillance

I also want to chime in with how valuable this resource has been to me & my family personally, and would also be willing to financially contribute on Patreon or similar.

**Reply by daniela-githubaccount** [reply]
Would love to see and include those data, but (at least someplace outside of the US) they are not necessarily available to the general public in near realtime.

**Comment by lazerwalker** [comment]
One other concern I've personally had recently is that the current calculations are (as far as I can tell) still using Delta-level data for variables like transmissibility that informally seem likely to be underestimates for the current strains. I don't know how much concrete data we have on current strains, but that also feels like an issue likely to lead to misleading guidance. 

(Count me as another coder with no stats background but who's happy to pick off tasks if there are ways to be helpful -- I'll poke at the backlog!)

**Reply by apiology** [reply]
Have you reviewed the 1/4/2022 changelog update?  https://www.microcovid.org/paper/changelog. That should help clarify what was and was not updated for at least the original Omicron variant.  More detailed and updated work is welcome!

**Comment by squeakrad123** [comment]
I am not a coder and don't understand what most of the terms you're talking about me. But I would be happy to contribute if there is someone else who has a Venmo or PayPal account. I rely on micro Covid almost every week and I would be really devastated if the site became unusable.

**Comment by whispersinfruit** [comment]
What skills, coding languages, etc would be helpful for this development?

**Reply by apiology** [reply]
Hey, all!  I touched on the technology part of the stack a bit in some other answers, but does anyone want to help @whispersinfruit with a more detailed answer to this question?

**Comment by Drumsticks49** [comment]
I don't have anywhere near to skills to help, but I'm seconding (thirding, fourthing) that I would gladly send money if money would help - like, enough to justify an engineer taking a day off work to update it, if that would help.

**Comment by ProfessorFalken00** [comment]
I don't have any of the technical skills needed to help, but I find MicroCovid very valuable and would be willing to contribute money to help keep it up to date. Hope that's helpful and thank you for your work.

**Comment by Tune-kt** [comment]
Just want to say I appreciate your site a lot : ) While I don't have much to offer, I think many would agree that it would be unfortunate if microcovid goes. 

I think that, even without access to high quality data, the tool is still valuable for estimating relative risks. And I think users can give their own estimation about covid prevalence in their locations (in the worst case, we can always just multiply official prevalence data by 10~100 and get a high caution result). I'm personally more interested to know whether the new variants have different attack rates, and whether there are updates about the effect of masks/ventilation, but hopefully information on these issues don't go obsolete as quickly as regional prevalence data.

**Reply by Tune-kt** [reply]
China's CDC recently published this record of a mass-spreading event, and it has got the attention of the covid community on twitter.
https://weekly.chinacdc.cn/en/article/doi/10.46234/ccdcw2022.209

I'm somewhat sceptical that there aren't other compounding factors that caused 30+ people to be infected by a jogger, but perhaps it means "exercising" is a pretty huge transmission risk factor.

**Comment by daniela-githubaccount** [comment]
Hi, 
first of all, thank you for your work so far. We believe there is a significant need for this app to stay online and updated with current scientific knowledge. 
We are a small team and we have been using your webapp and other tools to manage our covid risk. We also have been studying scientific literature, as well as the courses of Prof. Bazant's 
on Physics of Covid-19 Transmission on edX, now available on MITx 
https://mitxonline.mit.edu/courses/course-v1:MITxT+10.S95x/
and Prof. Babson's on Mathematics of Covid-19 at UC Davis  
https://www.math.ucdavis.edu/~babson/280-F22/index_280-F22.html 
We have been lucky as well as careful, and the three of us did not catch COVID so far. 

We are sorry to hear you are understaffed, we would be very happy to contribute, but we are not able to volunteer much, as we are financially beyond strapped at this time. 

Please let us know how we can help.

**Reply by apiology** [reply]
Interesting classes!  Are there takeaways from them you think could be applied to microCOVID?

**Comment by xbruge** [comment]
Just wanted to share that I've been referring to [this site](https://thoughtfulnz.quarto.pub/nzcovidreport/) for local prevalence updates, perhaps an option to allow users to find and add their own sources somehow could be a solution?

**Reply by apiology** [reply]
Yeah, having an input of some rough measure of local prevalence (maybe low/medium/high?) could be done.  What situations do you think this could be most helpful for?

**Comment by w-gitops** [comment]
My wife and I highly value this site, and I would love to help on the data pipeline (for starters).  i actually work for a company that makes a key tool for data analytics engineering and transformation, and this would be a meaningful passion project for me to keep my practical skills up to date.  If this sounds interesting, lets talk!  I’ll take a closer look at the repo in the meantime, as while I’m not a pro dev, I’m an armchair coder as well.  Thanks again for all this site does!

**Reply by apiology** [reply]
Sounds great!  Want to give some thought on what you'd like to hack on?  Once you have something in mind,  maybe [book some time](https://calendly.com/apiology/microcovid) with me and we can figure out how to make that happen together?

In addition to the above critical stuff, there are plenty of ideas hanging around in the discussion posts, issues and PRs that could use some help.  Other ideas are of course welcome!

**Comment by shawnbiesan2** [comment]
My partner and I use the site heavily and would love to help. Both professional developers, but not specifically python. Will start taking a look at the repo to see if there are areas we can help

**Reply by apiology** [reply]
Fantastic, @shawnbiesan2!  Good timing too - looks like our [prevalence pipeline is broken again](https://github.com/microCOVID/microCOVID/issues/1654), meaning numbers are now four days stale and growing staler.  

If you, @willsargent, @aidantjones, and/or anyone else wants to take a crack at this, I added some bread crumbs in the issue, including my scheduling link to pair via Zoom.  Happy to help you through the setup process and answer questions as you go!

**Comment by boxlike-supply** [comment]
I also make frequent use of Microcovid in daily decision-making and I'm extremely grateful for the original team who created it and have maintained it.  The data pipeline for global sources is a huge convenience, but the key feature for me is figuring out relative risk based on activity and circumstances. I'm glad to see volunteers stepping up to help, but if you have to make the unfortunate decision to deprecate some of the site, I would hope the risk framework can be preserved for use even if the prevalence calculations are not automatic anymore (basically making the "override location-based data the default".)

**Comment by mikaelaf** [comment]
I'm not a programmer or data science person, but I am a decently intelligent person with spare time who values the microcovid project a whole lot. So let me know if there's anything you want help with that a layperson might be able to do! 

Skills I do have that might applicable:
- graphic design
- html/css
- a small amount of marketing knowledge
- a larger amount of marketing experience
- writing/editing
- childcare (i babysit the children of someone who knows how to program? lol)

**Reply by apiology** [reply]
Happy to pair up or make connections when you see the right opportunity!  Just reach out and let me know what you'd like to take on.

**Reply by apiology** [reply]
There's an issue tag for documentation if you are looking for inspiration: https://github.com/microCOVID/microCOVID/labels/documentation%20%2F%20white%20paper

**Comment by mmailliw** [comment]
I have been using this site regularly to gauge risk, or at least as a heuristic.  As far as I can tell, the biggest thing that needs to be changed urgently is the ascertainment ratio:  playing around with parameters when I "Override location-based data" seems to show that the ratio of "adjusted prevalance" to "reported prevalance" is given as somewhere between 2:1 and 3:1.  However, the current ascertainment ratios seem to be much worse.

In particular, if you look at Figure 8.1 in [this briefing](https://www.healthdata.org/sites/default/files/covid_briefs/570_briefing_Washington.pdf?fbclid=IwAR029OLyM6viMk2CStzoKV2XBhzXZ4zuH1zINSfNurK7kXWf2XPD6FWE_pw), you will see that as of around two weeks ago, only about 5% of "estimated cases" were detected.  If this is the correct number, it corresponds to more like a 20:1 ratio of "adjusted prevalance" to "reported prevalance"!  (As a rough correction, I have therefore advised people to multiply microCOVID values out of this site by 8, as the ratio of 20:1 to 2.5:1.)

Wastewater data might be useful for modeling the ascertainment ratio... but the range [2, 3] for an ascertainment ratio does seem too narrow in light of Figure 8.1.

**Reply by daniela-githubaccount** [reply]
Thank you a lot for your comment, and I agree with your assessment. On the other hand I find microCOVID to overestimate the risk when the user is wearing (correctly) N99 and similar.

**Reply by lazerwalker** [reply]
@daniela-githubaccount Do you have citations for the efficacy of N99 masks that differs from the current sealed N95 or P100 options? That feels like that could be a reasonable PR if so.

**Reply by daniela-githubaccount** [reply]
@lazerwalker 

The current risk reduction factors are 
1) N95 / KN95 / FFP2 1/3x
2) N95 sealed 1/8x 
3) P100 respirator 1/20x 

It is not clear to me what is 1) a non-sealed N95. Does that mean incorrectly worn? 
Also, 3) seems to refer to full-face and half-face masks. I do not think there is a category for the N99, N100, P100 disposable (single use or multiple use) aka FFP3 (the difference in standard relates to oily aerosols that is not critical here). While most  standardization institutions refrain from giving numbers "it significantly protects..." the British say the N100 to provide at least 25x risk reduction for transmission of tubercolosis. I will post the label later or tomorrow. I would think covid-19 is comparable.
I expect N99 to be not far from N100; also, in N99 about half of the incoming external air comes from the seal, and for N100 it's most of it. If the user has not been fitted with the test hood etc.... there is hardly any difference. 
I would say the N99-N100-FFP3 are preferable to N95, especially if exposure is for long periods, but not overwhelmingly. And they are less comfortable. 

As for the full facepiece with P100, or the hood - another option that I don't see listed, that's a different story, because the risk is not homogeneous. The risk will be basically zero when correctly worn, and will be non zero when you take it off! What is the changing room like? How experienced the user is? Not everyone is a nuclear technician or hazard material handler. It is not easy to quantify the risk, I agree that 1/20 is a reasonable conservative estimate, but the risk is highly dishomogeneous. Also I am not sure if any people in 2022 (saw a few in 2020) really wear these in daily life. They are very noticeable and it takes a lot of determination to wear it regularly. There may also be police implications, i.e. getting stopped under suspicious of preparing a terrorist attack with biological or chemical or nuclear release. They may be a good choice when visiting someone with known covid, but even so, not very practical.

The half-face has the additional unknown of eyes, is the wearer using goggles? (if so why not a full-face?) Lab glasses? No eye protection?

**Reply by zerotrickpony** [reply]
+1 to the point that the mask multipliers for heavy masks are wrong for both audiences. It's an average set on a bimodal distribution: likely too conservative for correctly sealed and diligently worn masks, and likely not cautious enough for someone who wears the mask incorrectly or inconsistently.

**Reply by apiology** [reply]
Thanks, @mmailliw, and appreciate the citations!  I think a good next step would be for an issue to be filed on this where we can work out the specifics of the adjustment to be made.

(ditto for a separate issue on the masks, @daniela-githubaccount)

**Comment by mariehuynh** [comment]
I'd hate to see this deprecated and would like to help.  I'm an experienced software engineer and previously worked for LIGO.  I haven't used React specifically, so I'll need to do some tutorials and will need some time to ramp up before I contribute. I see that there are some issues tagged as "Good first issue". How do I get access to the Slack?

**Reply by apiology** [reply]
Thanks so much for the PR, @mariehuynh!  Just sent you an invite.

**Comment by bookwormlesbo** [comment]
hey! I don’t really know any details about software engineering, but i can say that this tool is so important to me and my community’s wellbeing so i’d love to do anything i can to save it. i’d be happy to spread the word about this on social media and if money is an issue, starting a GFM could be useful.

**Reply by apiology** [reply]
Yes, that suggestion has come up frequently.  Please see [here](https://github.com/microCOVID/microCOVID/discussions/1626#discussioncomment-4046309) for next steps you can do to make this happen!

**Comment by squeakrad123** [comment]
When I click on the ”here” it doesn’t take me to any other page. Is there something I should be looking for?Sent from my iPadOn Dec 3, 2022, at 4:15 AM, Vince Broz ***@***.***> wrote:﻿
Yes, that suggestion has come up frequently.  Please see here for next steps you can do to make this happen!

**Reply by apiology** [reply]
Here you go:

<img width="860" alt="image" src="https://user-images.githubusercontent.com/3681194/205457794-01644bf5-be83-4da8-b6ca-c0fc165989f8.png">

**Comment by squeakrad123** [comment]
So I sent Catherine a dollar with a request to let us know if there was any money or any other way to donate to the project. I’ll let you know what happens. But isn’t there anyone on the project now would be willing to give their PayPal address

**Reply by apiology** [reply]
OK--thanks for that!

If someone believes a donation model would raise enough to make microCOVID more sustainable, and wants to volunteer to do the leg work to make that happen, I'd be happy to brainstorm tactics--just [book some time](https://calendly.com/apiology/microcovid) with me.   I can handle any needed coding bits once we figure out a plan together.

p.s., @squeakrad123, would you mind using threads in the discussions area or creating a separate issue or discussion post for donation discussions?  You're creating a top-level item on an already very cluttered discussion post each time you reply.

**Comment by squeakrad123** [comment]
Sorry. I’ve never use this platform before and trying to do a bunch of things at onceSent from my iPhoneOn Dec 3, 2022, at 12:37 PM, Vince Broz ***@***.***> wrote:﻿
OK--thanks for that!
If someone believes a donation model would raise enough to make microCOVID more sustainable, and wants to volunteer to do the leg work to make that happen, I'd be happy to brainstorm tactics--just book some time with me.   I can handle any needed coding bits once we figure out a plan together.
p.s., @squeakrad123, would you mind using threads in the discussions area or creating a separate issue or discussion post for donation discussions?  You're creating a top-level item on an already very cluttered discussion post each time you reply.

**Comment by squeakrad123** [comment]
Actually you know what I am doing the best I can if I hear from Catherine  I’ll let you know, but otherwise I’m out.Sent from my iPhoneOn Dec 3, 2022, at 12:37 PM, Vince Broz ***@***.***> wrote:﻿
OK--thanks for that!
If someone believes a donation model would raise enough to make microCOVID more sustainable, and wants to volunteer to do the leg work to make that happen, I'd be happy to brainstorm tactics--just book some time with me.   I can handle any needed coding bits once we figure out a plan together.
p.s., @squeakrad123, would you mind using threads in the discussions area or creating a separate issue or discussion post for donation discussions?  You're creating a top-level item on an already very cluttered discussion post each time you reply.

**Comment by sunny-bak** [comment]
I would like to help with some coding and stat work. I may be limited in time I can put in the project but I would like to help in areas I can.

**Reply by apiology** [reply]
Fantastic!  In addition to the above critical stuff I wrote about above, there are plenty of ideas hanging around in the discussion posts, issues and PRs that could use some help.  Other ideas are of course welcome, and please reach out if I can help pair on things to help get started.

**Reply by apiology** [reply]
A critical thing that just popped up:

* https://github.com/microCOVID/microCOVID/issues/1695

Happy to pair with you or anyone else who wants to take this on - you can book some time with me here: https://calendly.com/apiology/microcovid

**Comment by daniela-githubaccount** [comment]
https://mms.isiaq.org//Calendar/moreinfo_responsive.php?org_id=ISIA&eventid=74477

ISIAQ 30th Anniversary Webinar: The long history of airborne infection transmission: why don’t we use the knowledge we have

Starting now

**Comment by apiology** [comment]
Hi all,

You may have seen this message:

<img width="331" alt="image" src="https://user-images.githubusercontent.com/3681194/208088274-f8656bc9-f0fe-461f-8337-474320df816f.png">

This means that the prevalence pipeline is once again broken and no changes are happening to microCOVID numbers over time.

There's information here about the failure here:

* https://github.com/microCOVID/microCOVID/issues/1695

We need your help to fix this.

I'm happy to pair to help someone if you don't know where to start - you can book some time with me here: https://calendly.com/apiology/microcovid

**Reply by apiology** [reply]
Thankfully this seems to have resolved itself.  Thanks to @xaqrox for taking a look at it!

**Comment by bermannoah** [comment]
Hello, I'd love to help with some coding work -- I'll take a look at the open issues but I use this project regularly and would like to help keep it online, if I can! :)

**Comment by zack112358** [comment]
Oh goodness, I'm glad that there's an upswell of community support here. I would like to pitch in some time; I'm a 10 year Python dev and I've done front-end too (though not React). Can you add me to the slack?

**Reply by apiology** [reply]
Sent, and thanks for booking some time!  I look forward to talking.

**Comment by yek3l** [comment]
Hello! Thank you so much for laying these steps out so clearly. I actually work in data quality / pipeline testing (primarily working in Python and SQL), and have really relied and benefited from the microcovid tool these last few years. I'd hate to see it go down, and would love to pitch in how I can. It might take me a bit to get up to speed on the existing pipelines, but I'd really like to help. Could you add me to the Slack as well? My email is ye.kellyann@gmail.com.

**Reply by apiology** [reply]
@yek3l: Sent - thanks for reaching out!

**Reply by apiology** [reply]
In the meantime, there's a `skill: backend engineering` tag you can search in issues for pipeline-related needs: https://github.com/microCOVID/microCOVID/issues?q=is%3Aissue+is%3Aopen+label%3A%22skill%3A+backend+engineering%22

**Comment by maxwelldb** [comment]
Hey, any need for documentation--as in software docs, not necessarily white papers? If so, I'd be happy to help with that, as well as a11y or usability work. 

Slack invites welcome. 👍 

Email: maxwell dot bridges at gmail dot com

**Reply by apiology** [reply]
Hi @maxwelldb!  Just sent you an invite.  

I'd suggest looking at the https://github.com/microCOVID/microCOVID/labels/documentation%20%2F%20white%20paper tag in the issues to see if anything pops out as appropriate. If you see something where you'd need some info to be able to proceed, could you call that out?  I tried to add a https://github.com/microCOVID/microCOVID/labels/skill%3A%20model%2Fresearch tag to indicate that we'll need some thinking in that area too - maybe those are things folks could find ways to team up on.  

In the meantime, happy to share what I know about any that seem promising--welcome!

**Comment by velocitybucket** [comment]
The Microcovid framework has been super useful and I wish everyone on the planet would use it because it is really the only rational way to think about COVID risk.

Having said that, it seems to me that BY FAR the most pressing issue of the day is the question of **the prevalence and severity of long COVID**. It's a difficult issue, because there doesn't seem to be great data. But without SOME handle on this question, why measure Microcovids? Someone who is not very old or immunocompromised might ask: what's the big deal about getting infected with COVID?

The genius of the Microcovid approach is to take what IS known, by looking at primary literature, and to make educated guesses, which are then turned into useful heuristics that can applied by anyone. It seems to me that applying this approach to understanding long COVID should be the highest priority goal of the Microcovid project.

Perhaps some researchers or bloggers or others outside of the Microcovid project have done something like this? Some topically relevant posts, like [this one](https://www.mattbell.us/delta-and-long-covid/), are VERY out of date. Does anyone know of anything more recent?

**Comment by squeakrad123** [comment]
as an old and immunocompromised person i value measuring microcovids. Dont make gratuitous assumptions     On Wednesday, January 25, 2023 at 11:22:55 AM PST, velocitybucket ***@***.***> wrote:  
 
 


The Microcovid framework has been super useful and I wish everyone on the planet would use it because it is really the only rational way to think about COVID risk.

Having said that, it seems to me that BY FAR the most pressing issue of the day is the question of the prevalence and severity of long COVID. It's a difficult issue, because there doesn't seem to be great data. But without SOME handle on this question, why measure Microcovids? Someone who is not very old or immunocompromised might ask: what's the big deal about getting infected with COVID?

The genius of the Microcovid approach is to take what IS known, by looking at primary literature, and to make educated guesses, which are then turned into useful heuristics that can applied by anyone. It seems to me that applying this approach to understanding long COVID should be the highest priority goal of the Microcovid project.

Perhaps some researchers or bloggers or others outside of the Microcovid project have done something like this? Some topically relevant posts, like this one, are VERY out of date. Does anyone know of anything more recent?

**Reply by velocitybucket** [reply]
I understand. I too value measuring microcovids. My point was that **other** people might challenge the value of measuring microcovids. That's why I wrote "Someone who is not very old or immunocompromised might ask: what's the big deal about getting infected with COVID?", rather than writing something like "I ask you: what's the big deal about getting infected with COVID?"

So I don't think I'm making any gratuitous assumptions. As I said, I wish everyone on the planet would use the Micrcovid framework.

**Comment by apiology** [comment]
Hey all - I think we've taken this thread as far as GitHub's discussion feature will allow.  I'm going to lock it, but feel free to start new discussion threads based on anything here.
